heuristics/dbic_bids.py

```diff
@@ -20,6 +70,56 @@ def create_key(subdir, file_suffix, outtype=('nii.gz', 'dicom'),
     return template, outtype, annotation_classes
 
 
+def md5sum(string):
+    """Computes md5sum of as string"""
+    m = hashlib.md5(string.encode())
+    return m.hexdigest()
+
+
+def fix_canceled_runs(seqinfo, accession2run=fix_accession2run):
+    """Function that adds cancelme_ to known bad runs which were forgotten"""
+    accession_number = get_unique(seqinfo, 'accession_number')
+    if accession_number in accession2run:
+        badruns = accession2run[accession_number]
+        badruns_pattern = '|'.join(badruns)
+        for i, s in enumerate(seqinfo):
+            match = re.match(badruns_pattern, s.series_id)
+            if match:
```

Comment:
Minor: so why bother creating match variable and not just use the re.match directly in if?